src/main/java/com/flightstats/hub/dao/TtlEnforcer.java

```diff
@@ -37,9 +37,10 @@ public void deleteFilteredPaths(String path, ChannelService channelService,
                     .map(String::toLowerCase)
                     .filter(channel -> !channelSet.contains(channel.toLowerCase()) && !channel.equals(LOST_AND_FOUND_DIR))
                     .forEach(dir -> {
-                        String dirPath = path + "/" + dir;
-                        log.info("removing dir without channel {}", dirPath);
-                        commander.runInBash("rm " + " -rf " + dirPath, 1);
+
+                        String command = String.format("find %s/%s -type f --mmin +360 -delete", path, dir);
```

Comment:
A few concerns:

- The path is not quoted, which is scary, but I guess it wasn't quoted before so this isn't a regression
- The previous rm -rf would also delete the directory, but this won't. I'm not sure if that matters.
- The previous rm -rf would delete all files; this only deletes files modified more than 6 hours ago.

It sounds like this change would:
- keep more files around
- not delete the directory

Is that intended?

Reply:
I used the script essentially as it is running in chron currently as a patch.
I think the older than minute marker was a failsafe only for the patch version and should probably be removed.
I think adding in an unlink or rm -rf on the directory itself after the  files are deleted would be good and would not run into the same resource issue.

Reply:
The one in cron looks like this:
```
# clean up read spoke for items outside the spoke window.
25 * * * * find /spoke/read -type f -mmin +360 -delete
```
which is for the entire spoke read cache. It looks like this change above would be doing that find individually for each directory. But I don't see how removing the old stuff with find becomes more efficient than removing it with rm -rf if we're going for an entire directory.

Is the case that this rm -rf is only ever called for spoke write cache, and not spoke read cache?

Maybe @lkemmerer can chime in as they added this to cron

Reply:
talked with Laurie, closing this for now because I replaced this command in the wrong spot

Reply:
Reopened. I think the changes should address the concerns above, lemme know.